### PR TITLE
Reworked README

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**

### CODE REVIEW

File: cad/fp-info-cache [hunk 1]
```diff
@@ -0,0 +1 @@
+0
```
Comment (staticintlucas) [inline cad/fp-info-cache]: Was this added by accident?

File: README.md [hunk 2]
```diff
@@ -1,56 +1,104 @@
...
+based on the same `ATMega32U4` chip and programmable using the **Arduino IDE** <br>
+or a keyboard specific firmware such as `TMK` or `QMK`.
+
+![Preview]
+
+*Of course it is perfectly usable for any other electronics projects as well.*
+
+---
+
+**⸢ [PCB] ⸥ ⸢ [Schematic] ⸥ ⸢ [Vendors] ⸥ ⸢ [Changelog] ⸥**
```
Comment (staticintlucas) [inline README.md]: Please keep Vendors and Changelog in the README, not as separate files. The README isn't *that* long and I would prefer these to be visible on the GitHub project page

File: README.md [hunk 3]
```diff
@@ -1,56 +1,104 @@
...
-The goldfish is a Pro Micro compatible microcontroller designed for use in mechanical keyboards, although it is perfectly suitable for any other hobby electronics project.
-It is based on the same ATMega32U4 chip as the original Pro Micro and can be programmed using the Arduino IDE, or keyboard specific firmware such as TMK or QMK.
+
+This **Pro Micro** compatible board designed for use in `Mechanical Keyboards`, <br>
+based on the same `ATMega32U4` chip and programmable using the **Arduino IDE** <br>
+or a keyboard specific firmware such as `TMK` or `QMK`.
+
+![Preview]
+
+*Of course it is perfectly usable for any other electronics projects as well.*
```
Comment (staticintlucas) [inline README.md]: Can you move this sentence above the preview image?

File: README.md [hunk 4]
```diff
@@ -1,56 +1,104 @@
...
+
+<!----------------------------------------------------------------------------->
+
 # Goldfish
-The goldfish is a Pro Micro compatible microcontroller designed for use in mechanical keyboards, although it is perfectly suitable for any other hobby electronics project.
-It is based on the same ATMega32U4 chip as the original Pro Micro and can be programmed using the Arduino IDE, or keyboard specific firmware such as TMK or QMK.
+
+This **Pro Micro** compatible board designed for use in `Mechanical Keyboards`, <br>
+based on the same `ATMega32U4` chip and programmable using the **Arduino IDE** <br>
+or a keyboard specific firmware such as `TMK` or `QMK`.
```
Comment (staticintlucas) [inline README.md]: Couple of things here:
* Please don't put parts of the sentence in **bold** or `code`, it makes it weird to read
* Don't use a HTML `<br>`, let GitHub's line wrap do it's thing
* I would split this into two sentences. I think it's a bit too long to work as one

```suggestion
Goldfish is a Pro Micro compatible board designed for use in mechanical keyboards.
It is based on the same ATMega32U4 chip and is programmable using the Arduino IDE or a keyboard specific firmware such as TMK or QMK.
```

File: README.md [hunk 5]
```diff
@@ -1,56 +1,104 @@
...
+[Vendors]: docs/Vendors.md
+[PCB]: cad
+
+[Green Pro Micro]: http://www.40percent.club/2017/09/green-pro-micro.html
+[License]: https://creativecommons.org/licenses/by-sa/4.0/
+
+[Alvaro]: https://github.com/Ariamelon/Alvaro
+[Comet]: https://github.com/vattern/comet
+[Elite-C]: https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4
+[Goldfish-clone-JLCPCB]: https://github.com/JeremyJStarcher/Goldfish-clone-JLCPCB
```
Comment (staticintlucas) [inline README.md]: Can you move these links to the bottom of the file? The Markdown source reads much better that way

File: README.md [hunk 6]
```diff
@@ -1,56 +1,104 @@
...
+
+- Two internal pins make the **USB** data lines easily accessible.
+
+- `USB-C` is more ***durable***, ***reversible*** and just *plain cool*.
+
+- `5` additional pins on the underside of the board.
+
+- Pin-compatible with the original **Pro Micro**.
+
+- ***Sturdier*** through-hole **USB** connector.
```
Comment (staticintlucas) [inline README.md]: Same thing I said earlier about excessive **bold** and `code` in sentences.

Also please use single spacing for the bullet points. It ends up being a bit more compact when rendered by GitHub, which I think looks better

File: README.md [hunk 7]
```diff
@@ -1,56 +1,104 @@
...
+
+## License
+
+**[Creative Commons Attribution Share-Alike 4.0 License][License]**.
+
+
+**⤷** Original `Arduino Mini` design by **Team Arduino**. <br>
+**⤷** Arduino `Pro Mini` design by **Spark Fun Electronics**. <br>
+**⤷** `Pro Micro` design by **Spark Fun Electronics**. <br>
+**⤷** `Goldfish` design by **Dr Derivative**.
```
Comment (staticintlucas) [inline README.md]: Please stick with regular bullet points here

**Review by staticintlucas — CHANGES_REQUESTED** [review]
Overall I think it's a nice rework to the README, but I've requested a couple of changes to some parts. Mainly I would prefer to keep everything in one file, rather that to split out the Vendor and Changelog sections.

Also, can you please use lowercase file names for Preview.png and Schematic.pdf? Just for consistency

File: cad/fp-info-cache (hunk 1 above)
Comment (ElectronicsArchiver) [reply cad/fp-info-cache]: Likely automatic EOF conversion by git | Windows ⬌ Linux

Comment (staticintlucas) [reply cad/fp-info-cache]: Can you revert it in that case?

Comment (ElectronicsArchiver) [reply cad/fp-info-cache]: Oh right, no that file is just generated by KiCAD, you will probably want to add it to the `.gitignore` file then.

**Review by staticintlucas — APPROVED** [review]

**Pull request merged**